app/domain/output_document/csv_renderer.rb

```diff
@@ -0,0 +1,25 @@
+require 'csv'
+
+class OutputDocument
+  class CSVRenderer
+    attr_accessor :output_document
+
+    FIELDS = %i(id format variant attendee_name attendee_address_line_1
+                attendee_address_line_2 attendee_address_line_3 attendee_town
+                attendee_county attendee_postcode lead guider_first_name
+                guider_organisation appointment_reference appointment_date
+                value_of_pension_pots income_in_retirement continue_working
+                unsure leave_inheritance wants_flexibility wants_security
+                wants_lump_sum poor_health)
+
+    def initialize(output_document)
+      @output_document = output_document
+    end
+
+    def render
+      CSV.generate(col_sep: '|') do |csv|
```

Comment:
Looks like `converters` is `nil` by default:

http://ruby-doc.org/stdlib-1.9.2/libdoc/csv/rdoc/CSV.html#DEFAULT_OPTIONS